jenkins-scripts/dsl/gz-collections.yaml

```diff
@@ -337,6 +337,83 @@ collections:
         nightly:
             distros:
               - jammy
+  - name: 'ionic'
+    libs:
+      - name: gz-cmake
+        major_version: 4
+        repo:
+          current_branch: main
+      - name: gz-tools
+        major_version: 2
```

Comment:
we're bumping tools right?

Reply:
No, as @scpeters  pointed out on slack, bumping gz-tools will prevent us from installing Ionic side by side with any older release, so the plan is to update gz-tools to build with either gz-cmake3 or gz-cmake4. I've updated the description in https://github.com/gazebo-tooling/release-tools/issues/1027

Reply:
ok plan sounds good